src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,77 @@
         </div>
     {% endif %}
 
+    <h2>Security Firewall</h2>
+
+    {% if collector.firewall %}
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name }}</span>
+                <span class="label">Name</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.securityEnabled ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Security enabled</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.stateless ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Stateless</span>
+            </div>
+            <div class="metric">
+                <span class="value">{{ include('@WebProfiler/Icon/' ~ (collector.firewall.allowsAnonymous ? 'yes' : 'no') ~ '.svg') }}</span>
+                <span class="label">Allow Anonymous</span>
+            </div>
+        </div>
+        {% if collector.firewall.securityEnabled %}
+            <table>
+                <thead>
+                    <tr>
+                        <th scope="col" class="key">Key</th>
+                        <th scope="col">Value</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <th>provider</th>
+                        <td>{{ collector.firewall.provider }}</td>
+                    </tr>
+                    <tr>
+                        <th>context</th>
+                        <td>{{ collector.firewall.context }}</td>
+                    </tr>
+                    <tr>
+                        <th>entry_point</th>
+                        <td>{{ collector.firewall.entryPoint }}</td>
+                    </tr>
+                    <tr>
+                        <th>user_checker</th>
+                        <td>{{ collector.firewall.userChecker }}</td>
+                    </tr>
+                    <tr>
+                        <th>access_denied_handler</th>
+                        <td>{{ collector.firewall.accessDeniedHandler }}</td>
+                    </tr>
+                    <tr>
+                        <th>access_denied_url</th>
+                        <td>{{ collector.firewall.accessDeniedUrl }}</td>
+                    </tr>
+                    <tr>
+                        <th>listeners</th>
+                        <td>{{ collector.firewall.listeners|yaml_encode }}</td>
+                    </tr>
+                </tbody>
+            </table>
+        {% endif %}
+    {% elseif collector.enabled %}
+        <div class="empty">
+            <p>There is no security firewall.</p>
+        </div>
+    {% else %}
+        <div class="empty">
+            <p>The security component is disabled.</p>
```

Comment:
I'm thinking to remove this `else` statement. As is, this message would appear in both `Security Token` and `Security Firewall` sections. Maybe should I move [the existing one](https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig#L46) in a higher position and to make it hide the two sections?